app/scripts/modules/netflix/pagerDuty/pagerDutySelectField.component.ts

```diff
@@ -0,0 +1,51 @@
+import {module} from 'angular';
+import PagerDutyReaderModule from './pagerDuty.read.service';
+
+export interface PagerDutyService {
+  name: string;
+  integration_key: string;
+}
+
+export class PagerDutySelectFieldController implements ng.IComponentController {
+  public pagerDutyServices: [PagerDutyService];
+  public servicesLoaded: boolean;
+
+  static get $inject() { return ['pagerDutyReader']; }
+
+  constructor(private pagerDutyReader: any) {}
```

Comment:
make `public` (it's implicit but we're explicitly adding access modifier to all methods and fields)